data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -740,27 +736,11 @@ def build_academy_data(
         axis=1,
     )
 
-    academies["Status"] = academies.apply(
```

Comment:
Is `Status` just never used throughout?

Reply:
From a code search I couldn't find any usages. Pipeline also ran successfully.